ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -242,7 +242,14 @@ module.exports = Marionette.LayoutView.extend({
         .done((data, textStatus, jqxhr) => {
             _this.model.set({
                 type: 'custom'
-            })
+            });
+            const preferences = _user.getQuerySettings();
+            if (preferences.get('template')) {
+                preferences.set('type', 'custom');
+            } else {
```

Comment:
❓ I'm not sure I understand the else here.  Any reason we can't always set it to custom?

Reply:
You can only set a custom form template as default. If there is no default i figured that text would be a sane fallback.